GitUI/HelperDialogs/FormStatus.Designer.cs

```diff
@@ -30,12 +35,45 @@ private void InitializeComponent()
             this.MainPanel.Padding = new System.Windows.Forms.Padding(0);
             this.MainPanel.Size = new System.Drawing.Size(549, 246);
             // 
+            // PasswordPanel
+            // 
+            this.PasswordPanel.Controls.Add(this.PasswordSubPanel);
+            this.PasswordPanel.Controls.Add(this.PasswordSend);
+            this.PasswordPanel.Anchor = (AnchorStyles.Left | AnchorStyles.Right);
+            this.PasswordPanel.Dock = System.Windows.Forms.DockStyle.Bottom;
+            this.PasswordPanel.Margin = new System.Windows.Forms.Padding(0);
+            this.PasswordPanel.Padding = new System.Windows.Forms.Padding(8);
+            this.PasswordPanel.Size = new System.Drawing.Size(549, 39);
+            this.PasswordPanel.Location = new System.Drawing.Point(0, 249);
+            this.PasswordPanel.TabIndex = 2;
+            // 
+            // PasswordSubPanel
+            // 
+            this.PasswordSubPanel.Anchor = (AnchorStyles.Left | AnchorStyles.Right);
+            this.PasswordSubPanel.Margin = new System.Windows.Forms.Padding(0);
+            this.PasswordSubPanel.Size = new System.Drawing.Size(443, 23);
+            this.PasswordSubPanel.Location = new System.Drawing.Point(12, 8);
+            // 
+            // PasswordSend
+            // 
+            this.PasswordSend.Anchor = AnchorStyles.Right;
+            this.PasswordSend.Location = new System.Drawing.Point(461, 8);
+            this.PasswordSend.MinimumSize = new System.Drawing.Size(75, 23);
+            this.PasswordSend.Name = "PasswordSend";
+            this.PasswordSend.Size = new System.Drawing.Size(75, 23);
+            this.PasswordSend.TabIndex = 1;
+            this.PasswordSend.Text = "Send";
```

Comment:
```suggestion
            this.PasswordSend.Text = "&Send";
```
or remove this Button if it is not used at all?

Reply:
As I explained when adding it, the button is used for mouse-only login. User flow:
- Click on "Show password" checkbox
- Drag password from another program and drop it into the input box
- Click "Send" button
(It's technically possible to drag and drop a password directly to ConEmu, but not to the EditBox-based one)

Having a keyboard mnemonic for a mouse-only flow doesn't make a lot of sense.

Additionally, neither of the other buttons, "Abort", "Ok", have mnemonics. So it would stand out like a sore thumb.

Finally: It is a good thing none of the buttons have mnemonics. Imagine if the dialog opens, the keyboard focus is on the ConEmu, and a user, seeing the mnemonic, naively tries to press the matching key to reach the control. Rather than moving to the control, the key would be sent to the server. This is a pretty high-caliber footgun.

Unless this is an absolute must, I believe it would be better if it remained without a mnemonic.

Reply:
> As I explained when adding it,

The PR description shall describe the entire change. Comments are just complements for the reasoning and for the discussion of the implementation.

> the button is used for mouse-only login.
> Having a keyboard mnemonic for a mouse-only flow doesn't make a lot of sense.

One cannot anticipate all ways how users will interact with the dialog.
Mnemonics are missing at many places yet. New controls shall be accessible.
(New translations should not become necessary just because we forgot to add mnemonics when adding controls.)

> Additionally, neither of the other buttons, "Abort", "Ok", have mnemonics. So it would stand out like a sore thumb.

Not really: `OK` is the Default button (`AcceptButton`) and `Abort` the Cancel button.
And that's the key for a clearer UX and a simpler implementation:
As long as the text box is focused, the `Send` button is the Default button (it must not close the dialog, of course). Then there is no need for class `TextBoxSubmit`.
Since the Default and `Enabled` states of `OK` and `Send` are toggled when the process has finished, the `Send` button does not need a mnemonic because it will correctly indicate that it can be triggered by `Enter`.

> Finally: It is a good thing none of the buttons have mnemonics. Imagine if the dialog opens, the keyboard focus is on the ConEmu, and a user, seeing the mnemonic, naively tries to press the matching key to reach the control. Rather than moving to the control, the key would be sent to the server. This is a pretty high-caliber footgun.

I disagree.
Interception of mnemonics or not is / should be a question of suitable ConEmu configuration.
(`git.exe` runs locally.)

Reply:
Changed the Send button to be a default button when the password input is focused. I can see the difference now.

Is this fine now, or does it still require a mnemonic?

Reply:
>>  the Send button does not need a mnemonic

>  does it still require a mnemonic?

No, not the button but both checkboxes, please. (Yes, I am ignoring the sticky focus of ConEmu for now.)

And I think "Send" is too short and too technical. From user's perspective it is rather `Enter input` or just `Input` or maybe `Send input`. @RussKie?

Reply:
Both checkboxes? Should I change the text on `KeepDialogOpen` as well?